src/commands/build/build.ts

```diff
@@ -404,7 +413,11 @@ data _null_;
 file filecode;
 ${content}\n
 run;
-${getWebServiceScriptInvocation(serverType, undefined, false, true)}
+${
+  serverType !== ServerType.Sasjs
+    ? getWebServiceScriptInvocation(serverType, undefined, false, true)
+    : ''
+}
```

Comment:
sure, should I also follow `viya` approach in case of `sasjs` server type here https://github.com/sasjs/cli/blob/8c30547b04b4a24d297298f94d1bf5a9f7ad506b/src/commands/build/build.ts#L205 ?

Reply:
yes, done